source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -230,6 +242,39 @@ content: |
            To use the Azure Key Vault, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
            version 1.1 or later in your application's environment.
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           var keyVaultClient = new MongoClient(_connectionString);
```

Comment:
as I understand this constant is not mentioned anywhere in this doc? If so, I would replace it on the string value